site/docs/developer_documentation/getting_started.md

```diff
@@ -0,0 +1,196 @@
+
+- [C/C++](https://marketplace.visualstudio.com/items?itemName=ms-vscode.cpptools);
+- [CMake language support](https://marketplace.visualstudio.com/items?itemName=twxs.cmake) and [CMake Tools](https://marketplace.visualstudio.com/items?itemName=gocarlos.cmake-tools).
+
+
+### Clone the repository & install the necessary dependencies
+
+Make sure that you've [cloned GQCP](#cloning-the-repository) and [installed GQCP's dependencies](#installing-dependencies). We recommend using the `gqcp_dev` conda environment, so the necessary dependencies and their paths will all be set correctly.
+
+
+### Setting up your C++ compiler
+
+We often use [Clang](https://clang.llvm.org) 7.x.y as our C++ compiler. On macOS, this can easily be installed using [MacPorts](https://www.macports.org).
+
+```bash
+port install clang-7.0
+```
+
+After the installation, you should create a new toolkit by editing the `cmake-tools-kits.json`-file. (To open this file, open up the VS Code Command Palette and look for `Edit user-local CMake kits`).
+
+```json
+{
+    "name": "Clang 7.1.0",
+    "compilers": {
+        "C": "/opt/local/bin/clang-mp-7.0",
+        "CXX": "/opt/local/bin/clang++-mp-7.0"
+    }
+}
+```
+
+> **Note:** The paths to your compilers may be different. You can look for them using the command `which clang++` or `which clang++-mp-7.0` for clang 7.x.y.
+
+After this is done, you can select Clang 7.1.0 as your default compiler in VS Code.
+
+
+### Set up your GQCP workspace
+
+After opening the GQCP folder in VS Code, select `File > Save Workspace As...`, and save your workspace. This will create a folder `.vscode` with a file `<workspace-name>.code-workspace` in it. Within this file, make sure that CMake Tools is forwarded the correct options, i.e. make sure that the `"settings"` key at least contains the following options.
+
+```json
+{
+    "settings": {
+        "cmake.configureSettings": {
+            "CMAKE_PREFIX_PATH": "${CONDA_PREFIX}/envs/gqcp_dev",
+            "CMAKE_CXX_COMPILER": "/opt/local/bin/clang++-mp-7.0",
+            "BUILD_TESTS": "TRUE",
+            "BUILD_PYTHON_BINDINGS": "TRUE",
+            "CMAKE_INSTALL_PREFIX": "${CONDA_PREFIX}/envs/gqcp_dev",
+            "PYTHON_EXECUTABLE": "${CONDA_PREFIX}/envs/gqcp_dev/bin/python"
+        }
+    }
+}
+```
+
+In order to use the C++ IntelliSense correctly, you'll also have to set the corresponding include paths. Find `C/C++: Edit Configurations (JSON)` through VS Code's Command Palette and open the configurations for your recently created workspace folder. In the `c_cpp_properties.json`-file that is opened, please make sure to update your configuration's `"includePath"`.
+
+
+```json
+{
+    "configurations": [
+        {
+            "includePath": [
+                "${CMAKE_PREFIX}/envs/gqcp_dev/include",
+                "${CMAKE_PREFIX}/envs/gqcp_dev/include/eigen3",
+                "${workspaceFolder}/gqcp/include"
+            ]
```

Comment:
```suggestion
            "includePath": [
                "${CMAKE_PREFIX}/envs/gqcp_dev/include",
                "${CMAKE_PREFIX}/envs/gqcp_dev/include/eigen3",
                "${CMAKE_PREFIX}/envs/gqcp_dev/share/cmake/pybind11
                "${workspaceFolder}/gqcp/include"
            ]
```

While trying to takcle an issue on gqcp I found out that I still had include errors when it came to pybind11. The line I added solved this for me. I don't know whether it was a personal issue or if it's useful to add it to the documentation though.

Reply:
@xdvriend The extra path you've mentioned contains no header filers, only contains CMake files. I'm not sure that adding this path was really the solution to the problem you had.